packages/smooth_app/lib/pages/scan/camera_image_preview.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:smooth_app/helpers/camera_helper.dart';
+import 'package:smooth_app/pages/scan/camera_controller.dart';
+
+/// Forked Widget from the [camera] library, but with a simpler Widget
+class CameraStreamPreview extends StatelessWidget {
```

Comment:
```suggestion
class SmoothCameraStreamPreview extends StatelessWidget {
```

So that it's clear that we have it locally

Reply:
Ok, I will change that @M123-dev.
The PR is right now in draft to let @AshAman999 test if a fix is ok